InAppUtils/InAppUtils.m

```diff
@@ -220,7 +231,7 @@ - (void)productsRequest:(SKProductsRequest *)request
                                       @"currencyCode": [item.priceLocale objectForKey:NSLocaleCurrencyCode],
                                       @"priceString": item.priceString,
                                       @"countryCode": [item.priceLocale objectForKey: NSLocaleCountryCode],
-                                      @"downloadable": item.downloadable ? @"true" : @"false" ,
```

Comment:
is this a breaking change or this was a bug?

Reply:
A bug that was probably just generating a warning. But unless you're on the mac, the `getter` has been `isDownloadable` since `iOS 6.0`